src/newgrf.cpp

```diff
@@ -4707,6 +4708,131 @@ static ChangeInfoResult AirportTilesChangeInfo(uint airtid, int numinfo, int pro
 	return ret;
 }
 
+/**
+ * Ignore properties for roadstops
```

Comment:
```suggestion
 * Ignore properties for roadstops.
```

Although the period seems to be used mixed at the end of sentences.

Reply:
See the other comments in this file and the wider codebase